projects/igniteui-angular/src/lib/test-utils/ui-interactions.spec.ts

```diff
@@ -79,15 +79,48 @@ export class UIInteractions {
      * Clicks an element - native or debug, by dispatching pointerdown, focus, pointerup and click events.
      * @param element - Native or debug element.
      */
-    public static clickElement(element) {
+    public static clickElement(
+        element,
+        horizontal: HorizontalAlignment = HorizontalAlignment.Left,
+        vertical: VerticalAlignment = VerticalAlignment.Top) {
```

Comment:
Really no need for these, those synthetic interactions aren't really used to test handlers that need a point and the components that do care for coordinates I assume already use some form of helper for those.